src/clientiface.cpp

```diff
@@ -209,6 +209,13 @@ void RemoteClient::GetNextBlocks (
 	s16 d_max_gen = std::min(adjustDist(m_max_gen_distance, prop_zoom_fov),
 		wanted_range);
 
+	s16 d_max = full_d_max;
+
+	// Don't loop very much at a time
+	s16 max_d_increment_at_time = 2;
```

Comment:
This was `full_d_max / 9 + 1` before, why is this different?

Reply:
This is the partial revert of #6539, which is now no longer needed.

Reply:
I tested that the loadtime is still proportional to the number of blocks and independent of the zoom distance.